l1-contracts/test/foundry/unit/concrete/state-transition/chain-deps/facets/Mailbox/ProvingL2LogsInclusion.t.sol

```diff
@@ -0,0 +1,299 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.24;
+
+import {MailboxTest} from "./_Mailbox_Shared.t.sol";
+import {L2Message, L2Log} from "contracts/common/Messaging.sol";
+import "forge-std/Test.sol";
+import {L2_L1_LOGS_TREE_DEFAULT_LEAF_HASH, L1_GAS_PER_PUBDATA_BYTE, L2_TO_L1_LOG_SERIALIZE_SIZE} from "contracts/common/Config.sol";
+import {L2_TO_L1_MESSENGER_SYSTEM_CONTRACT_ADDR, L2_BOOTLOADER_ADDRESS} from "contracts/common/L2ContractAddresses.sol";
+import {Merkle} from "contracts/state-transition/libraries/Merkle.sol";
+import {MurkyBase} from "murky/common/MurkyBase.sol";
+import {MerkleTest} from "contracts/dev-contracts/test/MerkleTest.sol";
+import {TxStatus} from "contracts/state-transition/chain-deps/facets/Mailbox.sol";
+import {Bridgehub} from "contracts/bridgehub/Bridgehub.sol";
+import {MailboxFacet} from "contracts/state-transition/chain-deps/facets/Mailbox.sol";
+
+contract MerkleTree is MurkyBase {
```

Comment:
Shall we place it in the dev-contracts?

Reply:
Fixed: used the one from merkle tree tests, which is exactly the same. Is it enough?